internal/contentenc/content.go

```
@@ -133,7 +136,9 @@ func (be *ContentEnc) DecryptBlock(ciphertext []byte, blockNo uint64, fileID []b
 	if err != nil {
 		tlog.Warn.Printf("DecryptBlock: %s, len=%d", err.Error(), len(ciphertextOrig))
 		tlog.Debug.Println(hex.Dump(ciphertextOrig))
-		return nil, err
+		if be.forceDecode == false {
+			return nil, err
+		}
 	}
 
 	return plaintext, nil
```

Please bubble up the error:

return plaintext, err